ggml.h

```diff
@@ -657,8 +667,8 @@ extern "C" {
 
     GGML_API void    ggml_print_backtrace(void);
 
-    GGML_API void    ggml_numa_init(void); // call once for better performance on NUMA systems
-    GGML_API bool    ggml_is_numa(void); // true if init detected that system has >1 NUMA node
+    GGML_API void       ggml_numa_init(uint32_t numa); // call once for better performance on NUMA systems
```

Comment:
What's the reason for the parameter to be an int instead of an `enum ggml_numa_strategies`?

Reply:
This was a mistake on my part.

Reply:
This has opened a can of worms on my end. I'll try to find a clean way to make that enum explicit across the main code and test/example cases. @slaren any advice would be appreciated!

Reply:
I've pushed all those changes up. Let me know if anything still looks off.

Reply:
this is still uint32_t and not an enum